pkg/util/grunning/enabled.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+// See grunning.Supported() for an explanation behind this build tag.
+//
+//go:build !((darwin && arm64) || freebsd || !bazel)
```

Comment:
Looks fine for now, but we should keep in mind whether providing a way to opt-out (via the Bazel command line) is desirable. I can't personally come up with a reason why opting out would be necessary.

Reply:
Opt-out of what, using the grunning library or using the patched go? Features that build on top of this will have a disabled mode since we still have platforms we don't support this patch.

Reply:
Opt-out of using the grunning library, I meant. Opting out of using the patched go won't be possible unless you get into the `WORKSPACE` file and start updating stuff.

Sounds good.